### PR TITLE
SIDM-5280 fix error message

### PR DESCRIPTION
**Pull request by nikola-naydenov-hmcts**
If statement was in the wrong code block

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/idam/web/AppController.java [hunk 1]
```diff
@@ -214,11 +213,11 @@ public ModelAndView upliftRegister(@ModelAttribute("registerUserCommand") @Valid
                     return new ModelAndView(REDIRECT_RESET_INACTIVE_USER, model);
                 }
 
-                if (ex.getStatusCode().equals(HttpStatus.CONFLICT)) {
-                    msg = "Your account is already activated";
-                } else {
-                    msg += "PIN user not longer valid";
-                }
+                msg += "PIN user not longer valid";
```
Comment (jburke-idam) [inline src/main/java/uk/gov/hmcts/reform/idam/web/AppController.java:216]:
```suggestion
                msg += "PIN user no longer valid";
```
Or maybe just: msg = "Please try your action again. PIN user no longer valid"

File: src/main/java/uk/gov/hmcts/reform/idam/web/AppController.java [hunk 2]
```diff
@@ -214,11 +213,11 @@ public ModelAndView upliftRegister(@ModelAttribute("registerUserCommand") @Valid
...
+            }
+
+            if (ex.getStatusCode().equals(HttpStatus.CONFLICT)) {
+                msg = "Your account is already active.";
```
Comment (jburke-idam) [inline src/main/java/uk/gov/hmcts/reform/idam/web/AppController.java:220]: not "activated"? (as before)

**Review by jburke-idam — APPROVED** [review]
Approved, but if we're changing the wording on one of the messages it would be nice to fix the spelling mistake in the other.

File: src/main/java/uk/gov/hmcts/reform/idam/web/AppController.java (hunk 2 above)
Comment (nikola-naydenov-hmcts) [reply src/main/java/uk/gov/hmcts/reform/idam/web/AppController.java:220]: it sounded stupid

File: src/main/java/uk/gov/hmcts/reform/idam/web/AppController.java (hunk 1 above)
Comment (nikola-naydenov-hmcts) [reply src/main/java/uk/gov/hmcts/reform/idam/web/AppController.java]: Yeah, Fixed it

Comment (nikola-naydenov-hmcts) [reply src/main/java/uk/gov/hmcts/reform/idam/web/AppController.java]: Its supposed to say Please try again incase of like a 500 or something, I think is the login anyway

**Pull request merged**